library/src/main/java/com/meetme/support/fonts/FontManager.java

```diff
@@ -43,10 +43,8 @@
     private static final FontManagerImpl IMPL;
 
     static {
-        // TODO: SDK_INT == VERSION_CODES.N
-        if (Build.VERSION.SDK_INT == Build.VERSION_CODES.M && Build.VERSION.PREVIEW_SDK_INT > 0) {
+        if (Build.VERSION.SDK_INT >= 24) {
```

Comment:
I agree with using version codes provided by framework and not worrying about preview releases. I already pulled in the source to support a project I was on last year. I'm supposed to jump back on that project some time next week and can update this. But hopefully @iainalba can get around to it before that.

Thanks for getting back to us @jhansche !